### DISCUSSION TITLE
smooth scrubbing with slider in SMPlayer?

### DISCUSSION THREAD

**Discussion by dpettig** [smplayer-dev/smplayer#442] [General]
I am reviewing numerous slow motion videos where the feature of interest occurs in a short segment. To quickly find and then carefully review the event, I need a player with both forwards and backwards frame step as well as smooth scrubbing using the slider. The only free player I've found with these features is the old version of Quicktime. I have found the frame step functions, but can the slider be set up for smooth frame-by-frame scrubbing in SMPlayer?

**Comment by beedell-roke** [comment]
> I need a player with both forwards and backwards frame step

@dpettig, I've asked about that at [`discussions/1301`](https://github.com/smplayer-dev/smplayer/discussions/1301#discussion-9284720).
